style.css

```diff
@@ -0,0 +1,181 @@
+:root {
+  --neutral: #55423d;
+  --base-100: #fffffe;
+  --neutral-focus: #121629;
+  --primary: #eebbc3;
+  --secondary: #b8c1ec;
+}
+
+* {
+  margin: 0;
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: "Poppins" sans-serif;
+  text-align: center;
+  background-color: var(--base-100);
+}
+h1 {
+  text-align: flex;
+}
+span {
+  color: orange;
+}
+.images-daisy {
+  border-radius: 50%;
+}
+
+h2,
```

Comment:
I believe those p tags you are using in about page? Because I cannot find them anywhere in index.html , but only `h1`. Regarding `h1` I would reccomend to use it only once in page or in different sections, it can be an exception. Here is for the why https://www.semrush.com/blog/h1-tag/